README.md

```diff
@@ -98,14 +98,14 @@ Our audacious goal: _Zed will do for data lakes what JSON did for APIs._
 
 ## Zed & Brim
 
-Zed and Brim have become our _research vehicle_ for exploring the data model..
+Zed and Brim have become our _research vehicle_ for exploring the data model.
 
 * Zed & Brim are open source (BSD license)
     * [github.com/brimdata/brim](https://github.com/brimdata/brim)
     * [github.com/brimdata/zed](https://github.com/brimdata/zed)
 * Search-like experience optimized for Zeek and Suricata
     * [Zeek](https://zeek.org/) - maps packets to contextual logs
-    * [Suricata](https://suricata.io/) - threat detections engine
+    * [Suricata](https://suricata.io/) - threat detection engine
```

Comment:
"detections" actually seems ok too, but I figured we might as well go with the word they use on the Suricata site. :man\_shrugging: